terraform/alerting/verification-server/alerts.tf

```diff
@@ -327,3 +327,44 @@ EOT
     google_logging_metric.stackdriver_export_error_count
   ]
 }
+
+# fast error budget burn alert
+resource "google_monitoring_alert_policy" "fast_burn" {
+  project      = var.monitoring-host-project
+  display_name = "Fast error budget burn"
+  combiner     = "OR"
+  enabled      = "true"
+  # create only if using GCLB, which means there's an SLO created
+  count = var.https-forwarding-rule == "" ? 0 : 1
+  conditions {
```

Comment:
Rewrite this into MQL?

Reply:
Because SLOs don't support MQL, I don't believe there's a way to use MQL to express error budget burn rate conditions.

Reply:
In that case, write filter in a multi-line string literal will at least make it easier to read:

```
filter = <<-EOT
select_slo_burn_rate("projects/${var.verification-server-project}/services/verification-server/serviceLevelObjectives/availability-slo", "3600s")
EOT
```
Note it should be `var.verification-server-project` instead of `var.monitoring-host-project`. The latter one is used later if we decide to host both key server and verification server metrics in a single "host project", but right now we don't use `var.monitoring-host-project` at all.

Reply:
Done, thank you.